ifrs17/Import/Importers.ipynb

```diff
@@ -287,6 +287,92 @@
             "execution_count": 0,
             "outputs": []
         },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "## Import and Execution with Writing Down Logs into the DB"
```

Comment:
As you write, this should go somewhere else, as this Importers NB should be closer to business than to IT.

Reply:
Moved now to a dedicated notebook in Utils